docs/guides/managing_state.md

```diff
@@ -0,0 +1,192 @@
+---
+id: guides-managing-state
+slug: /guides/managing-state
+title: Managing OpenTofu and Terraform State in Massdriver
+sidebar_label: Managing State
+---
+
+# Managing OpenTofu and Terraform State in Massdriver
+
+## Overview
+
+Massdriver now offers integrated state management for OpenTofu / Terraform through our HTTP State backend. This feature empowers teams to manage their infrastructure state directly within the platform, whether they're migrating existing setups from S3, Terraform Cloud, or Azure Blob Storage, or starting from scratch. Massdriver's state management is designed to simplify and streamline your workflow, providing flexibility and control.
+
+Managing Terraform / OpenTofu state is a critical part of any infrastructure as code workflow. Proper state management ensures that your infrastructure is tracked, changes are applied consistently, and conflicts are avoided. With Massdriver, you have the flexibility to choose what works best for you: leverage our platform to handle state management seamlessly if you prefer not to manage it yourself, or retain full control by managing your state wherever you please. Whether you want a hands-off experience or need to integrate with your existing setup, Massdriver adapts to your workflow.
+
+## Setting Up Your Environment
+
+To get started with state management in Massdriver, follow these steps:
+
+### 1. Configure Your Environment Variables
+
+Set up your environment variables for authentication and state management:
+
+```bash
+
+export TF_HTTP_USERNAME=${MASSDRIVER_ORG_SLUG}
+export TF_HTTP_PASSWORD=${MASSDRIVER_SERVICE_ACCOUNT_TOKEN}
+
+# Your package friendly ID available from your details panel (without the four character suffix -xxxx)
+export MASSDRIVER_PACKAGE_ID="YOUR-PACKAGE-SLUG"
+
+# Massdriver supports putting multiple IaC tools into the same "bundle". 
+# Put the name of the step your OpenTofu / Terraform is in here.
+export MASSDRIVER_PACKAGE_STEP_NAME="the-step-name-in-your-package"
+
+export TF_HTTP_ADDRESS="https://api.massdriver.cloud/state/${MD_PKG_SLUG}/${MASSDRIVER_PACKAGE_STEP_NAME}"
```

Comment:
`MD_PKG_SLUG` isn't exported. Did you mean `MASSDRIVER_PACKAGE_ID` above? I'd rename that to `MASSDRIVER_PACKAGE_SLUG` since `MASSDRIVER_PACKAGE_ID` refers to the UUID in multiple places (in provisioning and XO)

Reply:
This was a typo in typing it up.

Its not the UUID though. Thats an internal identifier (and I'd love it if we stopped using it in userland across the board because... its compersome ᵃᶠ) it will be the natural key `ecomm-staging-database` without the suffix.